lib/openai/http.rb

```diff
@@ -53,13 +57,13 @@ def to_json(string)
     # @param user_proc [Proc] The inner proc to call for each JSON object in the chunk.
     # @return [Proc] An outer proc that iterates over a raw stream, converting it to JSON.
     def to_json_stream(user_proc:)
```

Comment:
A better name for this would be `to_completion_stream` because once the completion JSON is parsed, it's not longer "JSON" -- it's a completion object.